lib/commands/WeatherCommand.js

```diff
@@ -16,7 +16,11 @@ function getWeather(context, zip) {
     })
 
     weather.current({ method: 'zip', ZIPcode: zip, units: 'imperial' }, function(err, data) {
-        message.reply('the current weather for ' + data.name + ' is ' + data.weather[0].description + ', :thermometer: temperature is ' + data.main.temp + ' degrees. :droplet: humidity ' + data.main.humidity + '%, :wind_blowing_face: wind speed: ' + data.wind.speed + ' mph.')
+        if(typeof data.main === 'undefined') {
+            message.reply('no data exists for that zip code.')
+        } else {
+            message.reply('the current weather for ' + data.name + ' is ' + data.weather[0].description + ', :thermometer: temperature is ' + data.main.temp + ' ˚F. :droplet: humidity ' + data.main.humidity + '%, :wind_blowing_face: wind speed: ' + data.wind.speed + ' mph.')
```

Comment:
Use [ES6 template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here to cut down on the verbosity, e.g. `` `foo ${bar} baz` `` instead of `'foo ' + bar + ' baz'`.